### PR TITLE
Standardized the usage doc for cf-pagination

### PR DESCRIPTION
**Pull request by jimmynotjim**
Updated usage documentation for cf-forms to match the formatting and structure of #341 and #347.

## Additions

- Table of Contents

## Changes

- Standardized headings and org of description text

## Testing

- Test on your local markup preview tool

## Review

- @KimberlyMunoz 
- @contolini 
- @cfpb/front-end-team-admin 

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: src/cf-pagination/usage.md [hunk 1]
```diff
@@ -1,17 +1,25 @@
-The cf-pagination component provides a responsive approach to multipage page navigation.
+The cf-pagination component provides a responsive approach to multipage page navigation for Capital Framework.
```
Comment (KimberlyMunoz) [inline src/cf-pagination/usage.md]: Add a line for dependencies and remove the dependencies section.

**Comment by jimmynotjim** [comment]
@KimberlyMunoz Ready for review again

File: src/cf-pagination/usage.md [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
+
+[`cf-core`](../cf-core), ['cf-buttons'](../cf-buttons), and ['cf-icons'](../cf-icons) components are all dependencies of this component."
```
Comment (KimberlyMunoz) [inline src/cf-pagination/usage.md]: Same comment as the other review, I think only cf-core is in the tickmarks.

File: src/cf-pagination/usage.md (hunk 1 above)
Comment (KimberlyMunoz) [inline src/cf-pagination/usage.md]: Fixed!

**Comment by KimberlyMunoz** [comment]
After that line is fixed, 🚢  it.

**Comment by jimmynotjim** [comment]
@KimberlyMunoz thanks for catching these and the others. All fixed.

**Comment by KimberlyMunoz** [comment]
:+1:

**Pull request merged**